crates/types/src/joints.rs

```diff
@@ -671,6 +671,20 @@ impl Joints<f32> {
         }
     }
 
+    pub fn from_iterator<I>(angles: I) -> Self
```

Comment:
Isn't there a trait that you could implement rather than defining an unrelated method? Why not [`FromIterator`](https://doc.rust-lang.org/std/iter/trait.FromIterator.html)?

Reply:
Done